rpcd/playbooks/gen-facts.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
This playbook seems to be mixing generic naming with a specific purpose. This playbook defaults to gathering facts for mons yet the name of the playbook and play to not make this clear.

Why has this playbook been created? Was it an attempt to remove code duplication?